modules/ROOT/pages/backup-restore/copy-database.adoc

```diff
@@ -210,6 +210,16 @@ The block format is the default format for all newly-created databases as long a
 For more information on the block format, see xref:database-internals/store-formats.adoc[Store formats].
 ====
 
+[NOTE]
+====
+In addition to the `--from-pagecache=<size>` option, a new option `--max-off-heap-memory` is introduced in Neo4j 2025.01.
+
+You can still use the `--from-pagecache` option to speed up the copy operation by specifying how much cache to allocate when reading the source.
```

Comment:
`--from-pagecache` controls the same value as `--max-off-heap-memory`, therefor it's no longer about how much cache to allocate when reading the source.

Reply:
@tinwelint, we say that `--max-off-heap-memory` is added in addition to the `--from-pagecache=<size>`, but it actually replaces it, which means `--from-pagecache=<size>` is removed, not deprecated. I am not sure I understand the process here.

Reply:
If it is deprecated, then it should be available both in the syntax and as an argument, no?

Reply:
The function of `--from-pagecache` changed and it got another more appropriate name too - this to not break scripts that already used `--from-pagecache`. I don't think we can tell PicoCLI (the lib we use to parse/run commands) to deprecate a certain option name.

Reply:
So, it is not deprecated. It is enhanced 😄. Thanks!